uncrustify.cfg

```diff
@@ -73,3 +73,6 @@ sp_before_ptr_star_func		= remove
 sp_before_byref_func		= remove
 
 eat_blanks_before_close_brace	= true
+
+pos_constr_comma			= lead_break
+pos_constr_colon			= lead_break
```

Comment:
The official uncrustify [docs](https://github.com/uncrustify/uncrustify/blob/master/documentation/htdocs/config.txt) don't list `lead_break` as a valid value for these 2 options.